contracts/cw1-whitelist/src/multitest.rs

```diff
@@ -92,4 +92,45 @@ mod test {
 
         assert_eq!(resp, FindMemberResponse { is_present: true });
     }
+
+    #[test]
+    fn contract_exec() {
+        let mut app = AppBuilder::new().build(|_, _, _| ());
```

Comment:
Donald use `AppBuilder` if you are using default `App`. Just use `App::default()`.